src/components/LoginPage/Login.tsx

```diff
@@ -66,220 +51,171 @@ export const LoginPage = function (props: Props) {
+                        alt=""
+                        width={300}
+                    />
+                </div>
+                <div className={fr.cx("fr-col-md-6")}>
+                    <form
+                        onSubmit={sendLogin}
+                        method="POST"
+                        id="login_form"
+                        style={{
+                            padding: "4rem",
+                            backgroundColor:
+                                fr.colors.decisions.background.alt.blueFrance
+                                    .default,
+                        }}
+                    >
+                        <fieldset
+                            className="fr-fieldset"
+                            id="login-1760-fieldset"
+                            aria-labelledby="login-1760-fieldset-legend login-1760-fieldset-messages"
+                        >
+                            <legend
+                                className="fr-fieldset__legend"
+                                id="login-1760-fieldset-legend"
+                            >
+                                <h3>Me connecter</h3>
+                            </legend>
+                            {!!props.messages.length && (
+                                <div
+                                    className="notification"
+                                    dangerouslySetInnerHTML={{
+                                        __html: props.messages,
+                                    }}
+                                ></div>
+                            )}
+                        </fieldset>
+                        <Input
+                            hintText="Email en @beta.gouv.fr ou email secondaire"
+                            label="Mon email"
+                            nativeInputProps={{
+                                type: "email",
+                                placeholder: "prenom.nom@beta.gouv.fr",
+                                onChange: (e) => setEmail(e.target.value),
+                                required: true,
+                            }}
+                            state={formErrors ? "error" : "default"}
+                            stateRelatedMessage={formErrors}
+                        />
+                        <ButtonsGroup
+                            buttons={[
+                                {
+                                    children: isSaving
+                                        ? "Envoi du lien de connexion..."
+                                        : "Recevoir le lien de connexion",
+                                    onClick: () => {},
+                                    disabled: isSaving,
+                                    type: "submit",
+                                },
+                            ]}
+                        />
+                        <hr />
+                        <h4 className={fr.cx("fr-mb-1w")}>Besoin d'aide ?</h4>
```

Comment:
Pas sûr de moi du tout pour cette syntaxe 😅
```suggestion
                        <h3 className={fr.cx("fr-mb-1w, fr-h4")}>Besoin d'aide ?</h3>
```